2021q2/bsddialog.adoc

```diff
@@ -1,32 +1,30 @@
 == BSDDialog - TUI Widgets
 
-Link: link:https://gitlab.com/alfix/bsddialog[Project repository]  
-Link: link:https://man.freebsd.org/dialog/1[Manual dialog utility]  
-Link: link:https://man.freebsd.org/dialog/3[Manual dialog library]  
+Link: link:https://gitlab.com/alfix/bsddialog[BSDDialog Project]  
+link: link://https://invisible-island.net/dialog[Dialog Project]
```

Comment:
Spurious // after 2nd "link:", and 1st one should start with uppercase

Reply:
fixed